### PR TITLE
Added Marathi and Hindi languages.

### PR DESCRIPTION
**Pull request by bantya**
Added **Marathi** and **Hindi** Indic languages.

😀

### CODE REVIEW

File: src/i18n/hi/index.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  en: {
```
Comment (yanfali) [inline src/i18n/hi/index.js]: change the language code please

File: src/i18n/mr/index.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  en: {
```
Comment (yanfali) [inline src/i18n/mr/index.js]: change the language code please

**Comment by yanfali** [comment]
@bantya please lint these files

File: src/i18n/mr/index.js (hunk 2 above)
Comment (bantya) [reply src/i18n/mr/index.js]: Changed it.

File: src/i18n/hi/index.js (hunk 1 above)
Comment (bantya) [reply src/i18n/hi/index.js]: Changed it.

**Comment by bantya** [comment]
Late night coding sux. 😖😂
Did all what you asked for (language codes, 2 missing entries and lint). Thanks for pointing it out.

😀

**Comment by noroadsleft** [comment]
@bantya Any chance you could review https://github.com/qmk/qmk.fm/pull/87 for us? It'd be greatly appreciated.

**Comment by noroadsleft** [comment]
Resolved merge conflict via https://github.com/qmk/qmk_configurator/pull/547/commits/789dc630da9b21c5f9555478dd7a1191d18bc5ba.

File: src/i18n/hi/index.js [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      help: {
+        label: 'जादूगर को बुलाओ'
+      },
+      print: { ...print },
+      tester: { ...tester },
+      keyboard: {
+        label: 'कीबोर्ड'
+      },
+      layout: {
+        label: 'नक्शा'
```
Comment (abhay8nitt) [inline src/i18n/hi/index.js]: layout: अभिन्यास

File: src/i18n/hi/index.js [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        }
+      },
+      errors: {
+        invalidQMKKeymap:
+          'क्षमा करें, यह एक मान्य QMK कीमैप फ़ाइल प्रतीत नहीं होती है।',
+        kbfirmwareJSONUnsupported:
+          'क्षमा करें, QMK विन्यासकर्ता kbfirmware JSON फ़ाइलों को आयात करने का समर्थन नहीं करता है।',
+        unknownJSON: 'क्षमा करें, यह QMK कीमैप फ़ाइल प्रतीत नहीं होती है।',
+        unsupportedBrowser:
+          'आप एक गैर-समर्थित ब्राउज़र का उपयोग कर रहे हैं। इस्तेमाल करें'
```
Comment (abhay8nitt) [inline src/i18n/hi/index.js]: can use असमर्थित

**Comment by yanfali** [comment]
@bantya going to be closing this as it's no longer mergeable. Thanks.

**Pull request closed without merging**